### PR TITLE
[services] Add cloudprofile.agent role to service accounts in terraform

### PR DESCRIPTION
**Pull request by jigold**

### CODE REVIEW

File: infra/gcp-broad/main.tf [hunk 1]
```diff
@@ -507,7 +510,10 @@ resource "google_storage_bucket_iam_member" "testns_batch_bucket_admin" {
 module "ci_gsa_secret" {
   source = "./gsa"
   name = "ci"
-  project = var.gcp_project
+  project = var.gcp_project,
```
Comment (daniel-goldstein) [inline infra/gcp-broad/main.tf:513]:
```suggestion
  project = var.gcp_project
```

File: infra/gcp/main.tf [hunk 2]
```diff
@@ -487,13 +490,19 @@ resource "google_storage_bucket_iam_member" "testns_batch_bucket_admin" {
 module "ci_gsa_secret" {
   source = "./gsa_k8s_secret"
   name = "ci"
-  project = var.gcp_project
+  project = var.gcp_project,
```
Comment (daniel-goldstein) [inline infra/gcp/main.tf:493]:
```suggestion
  project = var.gcp_project
```

File: infra/gcp/main.tf [hunk 3]
```diff
@@ -487,13 +490,19 @@ resource "google_storage_bucket_iam_member" "testns_batch_bucket_admin" {
...
+  iam_roles = [
+    "cloudprofile.agent"
+  ]
 }
 
 module "testns_ci_gsa_secret" {
   source = "./gsa_k8s_secret"
   name = "testns-ci"
-  project = var.gcp_project
+  project = var.gcp_project,
```
Comment (daniel-goldstein) [inline infra/gcp/main.tf:502]:
```suggestion
  project = var.gcp_project
```

File: infra/gcp-broad/main.tf [hunk 4]
```diff
@@ -458,6 +458,7 @@ module "auth_gsa_secret" {
   iam_roles = [
     "iam.serviceAccountAdmin",
     "iam.serviceAccountKeyAdmin",
+    "cloudprofile.agent"
```
Comment (daniel-goldstein) [inline infra/gcp-broad/main.tf:461]: This also needs to be on the test auth account, right?

**Review by daniel-goldstein — CHANGES_REQUESTED** [review]
Let me know if you need anything to apply these changes. This is a purely additive change so I'd rather apply it before merging so we know it's correct.

**Comment by jigold** [comment]
I'm not sure how to apply it anymore. Can we do it together on zoom at some point this week?

**Comment by daniel-goldstein** [comment]
Sure thing! Maybe thursday afternoon at some point? Feel free to grab something on my calendar.

**Review by daniel-goldstein — APPROVED** [review]

**Pull request merged**